lasagne/objectives.py

```diff
@@ -70,8 +95,9 @@ def __init__(self, input_layer, loss_function=mse):
         self.input_layer = input_layer
         self.loss_function = loss_function
         self.target_var = T.matrix("target")
+        self.mask_var = None
```

Comment:
It should be possible to set this from the constructor, otherwise it should be omitted.